ops-devnet/docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
 
   l2:
     pid: host # allow debugging
-    image: kromanetwork/geth:v0.4.4
+    image: kromanetwork/geth:tge
```

Comment:
Why did you change it?

Reply:
This PR requires a version of geth with the burgundy changes.

Reply:
FYI, this should change when geth is released.